mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -32,6 +38,14 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 	METHOD method_1005 getCenter ()Lnet/minecraft/class_243;
 	METHOD method_1006 contains (Lnet/minecraft/class_243;)Z
 		ARG 1 vec
+	METHOD method_1007 traceCollissionSide (Lnet/minecraft/class_238;Lnet/minecraft/class_243;[DLnet/minecraft/class_2350;DDD)Lnet/minecraft/class_2350;
+		ARG 0 box
+		ARG 1 approachVector
```

Comment:
There may be a more appropriate term than "approach". Does `intersectingVector` make sense in this context ?

Reply:
`intersectionVector` works better, yes.